test/package_sanity_tests.jl

```diff
@@ -0,0 +1,6 @@
+using Aqua
+
+@testset "Code quality" begin
+	Aqua.test_all(Agents, ambiguities = false, unbound_args = false)
```

Comment:
Still need to understand how to solve the `unbound_args` problems (but they seem innocuous nonetheless)

Reply:
Mmmh not an easy task, since they are innocuous I think it is okay like this